### PR TITLE
Brk management cli

### PR DESCRIPTION
**Pull request by michalxo**
Simple rename of function (jolokia is a management client).
Artemis also has two internal messaging clients available from CLI as `<instance>/bin/artemis send/client/receive`..

### CODE REVIEW

File: messaging_components/services/__init__.py [hunk 1]
```diff
@@ -39,6 +41,10 @@ def create_service(executor: Executor, service_name: str=None, **kwargs) -> Serv
             container_name = None
             if isinstance(executor, ExecutorContainer):
                 container_name = executor.container_name
+            elif isinstance(executor, ExecutorAnsible):
```
Comment (fgiorgetti) [inline messaging_components/services/__init__.py]: @michalxo This is a less restrictive condition, and so it should be the last one. If you put it here, then the other one (last elif below) will never get executed.

File: messaging_components/services/service_dummy_broker.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    def __init__(self, name: str, executor: Executor, **kwargs):
+        super().__init__(name, executor)
+        self.name = "artemis-service"
+
+        self.ansible_host = kwargs.get("ansible_host", "localhost")
+        self.service_default_port = kwargs.get("artemis_port", "61616")
+        self.service_web_port = kwargs.get("broker_web_port", "8161")
+        self.service_path = posixpath.join(kwargs.get("broker_path"), "bin", "artemis-service")
+        self.service_username = kwargs.get("broker_service_user", "jamq")
```
Comment (fgiorgetti) [inline messaging_components/services/service_dummy_broker.py]: @michalxo One important thing. Remember to add all the new variables to the document that describes variables that can be used with messaging components.

Comment (michalxo) [reply messaging_components/services/service_dummy_broker.py]: These variables are temporary only. Once I will make cluster work, there will be only 1 variable `data_file_path`. So this was just for testing purposes.

Comment (michalxo) [reply messaging_components/services/service_dummy_broker.py]: https://docs.google.com/document/d/1sCynB4DXNqNIG1cjiLFnsQMR1XZGI7w9pyfn-K-hWyk/edit# updated

File: messaging_components/virtual/broker_cluster/__init__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from iqa_common.ansible.ansible_inventory import AnsibleVirtualComponent
```
Comment (fgiorgetti) [inline messaging_components/virtual/broker_cluster/__init__.py]: @michalxo I do not see the AnsibleVirtualComponent in iqa_common (and there is no PR for it as well).

**Comment by fgiorgetti** [comment]
@michalxo I noticed that in messaging_abstract some of the Service methods have changed and have an extra argument: "wait_for_messaging=False". The other existing implementations of Service must also be modified to include it, or they will not be compliance to the expected implementation.

File: messaging_components/virtual/broker_cluster/__init__.py (hunk 3 above)
Comment (michalxo) [reply messaging_components/virtual/broker_cluster/__init__.py]: I see, it's part of `iqa_common`. I am going to commit it now.

**Comment by michalxo** [comment]
> @michalxo I noticed that in messaging_abstract some of the Service methods have changed and have an extra argument: "wait_for_messaging=False". The other existing implementations of Service must also be modified to include it, or they will not be compliance to the expected implementation.

Yes, I am open for discussion on how to resolve it best. ;) If we want some `wait` there or not.

File: messaging_components/virtual/broker_cluster/__init__.py (hunk 3 above)
Comment (michalxo) [reply messaging_components/virtual/broker_cluster/__init__.py]: https://github.com/rh-messaging-qe/iqa_common/pull/12

**Pull request merged**